packages-ui/roosterjs-react/lib/contextMenu/menus/createImageEditMenuProvider.tsx

```diff
@@ -188,6 +203,14 @@ function shouldShowImageEditItems(editor: IEditor, node: Node) {
     return safeInstanceOf(node, 'HTMLImageElement') && node.isContentEditable;
 }
 
+function getImageFromTarget(target: Node) {
```

Comment:
which scenario triggers this code? Maybe we should consider add it to image selection code and relay only in 'getSelectionRangeEx'

Reply:
For roosterjs-react, I think it is ok to only support image selection.